tests/lava/lib/dl/netx/test_hdf5.py

```diff
@@ -129,6 +129,39 @@ def test_tinynet(self) -> None:
             f'Error was {error}.'
         )
 
```

Comment:
similarly here, the tests below is beyond the scope of the PR, check you intended for it to be in this PR,.